comid/cond_endorse_series_triple_test.go

```diff
@@ -4,44 +4,659 @@
 package comid
 
 import (
+	"encoding/json"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"github.com/veraison/corim/extensions"
 )
 
-func Test_CondEndorseSeries_NewCondEndorseSeriesTriples_OK(t *testing.T) {
+func Test_CondEndorseSeriesTriple_NewCondEndorseSeriesTriples_OK(t *testing.T) {
 	c := NewCondEndorseSeriesTriples()
 	require.NotNil(t, c)
+	assert.True(t, c.IsEmpty())
 }
 
-func Test_CondEndorseSeries_Valid_NOK(t *testing.T) {
+func Test_CondEndorseSeriesTriple_Add_Valid(t *testing.T) {
 	c := NewCondEndorseSeriesTriples()
-	expectedErr := "error at index 0: stateful environment validation failed: environment validation failed: environment must not be empty"
-	series := &CondEndorseSeriesTriple{}
+	// Create a valid triple
+	triple := &CondEndorseSeriesTriple{
+		Condition: ValueTriple{
+			Environment: Environment{
+				Class: &Class{
+					ClassID: MustNewUUIDClassID(TestUUID),
+				},
+			},
+		},
+		Series: *NewCondEndorseSeriesRecords(),
+	}
+
+	c.Add(triple)
+	assert.True(t, c.IsEmpty() == false)
+	err := c.Valid()
+	assert.Contains(t, err.Error(), "no measurement entries")
```

Comment:
you can use `assert.ErrorContains` directly (here and in many other places)

Reply:
It seems that assert.ErrorContains is still NOT used on line 38, while the comment got resolved?